pkg/model/types/types.go

```diff
@@ -16,15 +16,28 @@ limitations under the License.
 
 package types
 
+type ModelType int
 type ModelOutputType int
 
 var (
 	ModelOutputTypeConverter = []string{
```

Comment:
Now obsoleted. 
We are now having only `AbsPower` and `DynPower`.
Weight is no more used via estimator sidecar.

Reply:
Not sure if I fully understood this, what about the linear regression model type?
Can you fix this in another PR?